lib/rucio/web/rest/flaskapi/v1/replicas.py

```diff
@@ -1739,8 +1807,13 @@ def blueprint(with_doc=False):
     bp.add_url_rule('/bad/dids', view_func=bad_replicas_dids_view, methods=['post', ])
     replicas_rse_view = ReplicasRSE.as_view('replicas_rse')
     bp.add_url_rule('/rse/<rse>', view_func=replicas_rse_view, methods=['get', ])
+
     bad_replicas_view = BadReplicas.as_view('bad_replicas')
     bp.add_url_rule('/bad', view_func=bad_replicas_view, methods=['post', ])
+
+    quarantine_replicas_view = QuarantineReplicas.as_view('quarantine_replicas')
+    bp.add_url_rule('/quarantine', view_func=quarantine_replicas_view, methods=['post', ])
+
```

Comment:
Should a rule also be added in the `if not with_doc:` block below? I don't know the reason for the `if`, so I can't tell myself.